internal/report/shortnames_test.go

```diff
@@ -32,3 +32,29 @@ func TestShortNames(t *testing.T) {
 		})
 	}
 }
+
+func TestFileNameSuffixes(t *testing.T) {
+	type testCase struct {
+		name string
+		in   string
+		out  []string
+	}
+	test := func(name, in string, out ...string) testCase {
+		return testCase{name, in, out}
+	}
+
+	for _, c := range []testCase{
+		test("empty", "", "."),
```

Comment:
Is "." expected here? I'm worried a little we'll show this dot in the UI when the profile does not have filenames (e.g. binary had symbols but no debug info).

Reply:
It is an artifact of the implementation leaking through.

I can fix this in a subsequent change. Any suggestions for a better thing to display. Maybe "?" if neither func/file is available?

Reply:
At a minimum I'd probably just leave it empty for now. I.e. just not turn empty strings into the dot. For the "?" - maybe, not sure, would need to look at it first. I believe we also use "unknown" and "<unknown>" placeholders in some places, if we'd want to start using "?" I think three different ones might be too many :)